stepengine-api/src/main/java/org/tweetwallfx/stepengine/api/StepEngine.java

```diff
@@ -182,6 +182,12 @@ public Object get(final String key) {
             return properties.get(key);
         }
 
+        public <T> T get(final String key, final Class<T> clazz) {
+            @SuppressWarnings("unchecked")
+            final T value = (T) properties.get(key);

```

Comment:
Put suppress warnings annotation to method and change to direct returning the getted value to avoid temporary store the return value.